### PR TITLE
Onboard Viva Access Management Schema

### PR DESCRIPTION
**Pull request by snehitgajjar**
Please review @lindalu-MSFT

### CODE REVIEW

File: office-365-management-api/office-365-management-activity-api-schema.md [hunk 1]
```diff
@@ -215,6 +216,9 @@ This article provides details on the Common schema as well as service-specific s
 |230|TeamsUpdates|Teams Updates App Events.|
 |237|DefenderExpertsforXDRAdmin|Microsoft Defender Experts Adminstrator action events.|
 |231|PlannerRosterSensitivityLabel|Microsoft Planner roster sensitivity label events.|
+|251|VfamCreatePolicy|Viva Access Management policy create event.|
```
Comment (edwin-king) [inline office-365-management-api/office-365-management-activity-api-schema.md:219]: May want to make `event` plural to match the other descriptions.

File: office-365-management-api/office-365-management-activity-api-schema.md [hunk 2]
```diff
@@ -2233,3 +2237,47 @@ The audit records for events related to Purview Governance use this schema (in a
 |OldValue |JObject |No |The old value of the object before change.|
 |NewValue |JObject |No |The New/current value of the object after change.|
 
+## Viva Access Management
+
+Viva Access Management comprises of the schema related to  policy management for various viva features. It includes action performed by the tenant IT Admin to create, modify, and delete the policy for various viva features. It uses t the [Common schema](#common-schema) with the following record types.
```
Comment (edwin-king) [inline office-365-management-api/office-365-management-activity-api-schema.md:2242]: Extra space between `to` and `policy management`

Comment (edwin-king) [inline office-365-management-api/office-365-management-activity-api-schema.md:2242]: Suggestion: "It includes actions" rather than "It includes action"

Comment (edwin-king) [inline office-365-management-api/office-365-management-activity-api-schema.md:2242]: Suggestion: Capitalize Viva

Comment (edwin-king) [inline office-365-management-api/office-365-management-activity-api-schema.md:2242]: extra `t` in "It uses t the"

File: office-365-management-api/office-365-management-activity-api-schema.md [hunk 3]
```diff
@@ -2233,3 +2237,47 @@ The audit records for events related to Purview Governance use this schema (in a
...
+|AddedIdentities|Collection(Edm.String)|Yes| List of user/group OIDs that are part of the created policy
+
+### VfamUpdatePolicy record type
+
+|**Properties**|**Type**|**Mandatory?**|**Description**|
+|:-----|:-----|:-----|:-----|
+|ModuleId|Edm.String|Yes|Viva module Id for which the policy is being created.|
+|FeatureId|Edm.String|Yes|Viva feature Id for which the policy is being created. feature is part of module.|
+|PolicyId|Edm.String|Yes| Policy Id of newly created policy for above module and feature ids.|
+|IsEnabeled|Edm.Boolean|Yes| Status showing if the policy is enabled or disabled for the given feature.|
```
Comment (edwin-king) [inline office-365-management-api/office-365-management-activity-api-schema.md:2269]: Typo in `IsEnabled` Also the policies themselves aren't enabled or disabled right? May want to clarify the desciption

File: office-365-management-api/office-365-management-activity-api-schema.md [hunk 4]
```diff
@@ -2233,3 +2237,47 @@ The audit records for events related to Purview Governance use this schema (in a
...
+|AddedIdentities|Collection(Edm.String)|Yes| List of newly added user/group OIDs that are part of the policy update|
+|RemovedIdentities|Collection(Edm.String)|Yes| List of removed user/group OIDs that are part of the policy update|
+|ModifiedFields|CollectionSelf.[AllowedModifiedFields](#enum-type-allowedmodifiedfields)|Yes| List of the fields that were modified as a part of the policy update.
+
+### VfamDeletePolicy record type
+
+|**Properties**|**Type**|**Mandatory?**|**Description**|
+|:-----|:-----|:-----|:-----|
+|ModuleId|Edm.String|Yes|Viva module Id for which the policy is being created.|
+|FeatureId|Edm.String|Yes|Viva feature Id for which the policy is being created. feature is part of module.|
```
Comment (edwin-king) [inline office-365-management-api/office-365-management-activity-api-schema.md:2279]: Suggestion: Some descriptions are ended in periods while others are not

**Review by edwin-king — COMMENTED** [review]
Added some suggestions

**Comment by snehitgajjar** [comment]
@lindalu-MSFT Please review.

**Review by lindalu-MSFT — APPROVED** [review]

**Comment by lindalu-MSFT** [comment]
@snehitgajjar I merged to live just now, should be available on production shortly!

**Pull request merged**